cluster/ubuntu/deployAddons.sh

```diff
@@ -41,15 +41,15 @@ function init {
 
 function deploy_dns {
   echo "Deploying DNS on Kubernetes"
-  sed -e "s/\\\$DNS_DOMAIN/${DNS_DOMAIN}/g" "${KUBE_ROOT}/cluster/addons/dns/skydns-rc.yaml.sed" > skydns-rc.yaml
```

Comment:
We should also fix the sed file to not use `$DNS_DOMAIN` but `VAR_DNS_DOMAIN` or `__DNS_DOMAIN` or something, so we don't have to double-escape like this...